index.html

```diff
@@ -10,14 +10,21 @@
 
     <div class="header">
         <div>
+          <div class="nav-cover">
             <nav>
                 <div class="logo"><a href="">ECHEZONA</a></div>
                 <a href="#" class="toggle-button">
                     <svg class="bar" width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
                     <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="#6070FF"/>
                     </svg>
                 </a>
+                <ul id="hamburger-menu">
+                  <li class="nav-links"><a href="#portfolio">Portfolio</a></li>
+                  <li class="nav-links"><a href="#about">About</a></li>
+                  <li class="nav-links"><a href="#contact">Contact</a></li>
+              </ul>
             </nav>
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, each of your projects section should be responsive to the mobile view breakpoint starting from 767px and below according to the project requirements.

There should not be any distortion at any breakpoint from 767px and below. At this moment your project is not properly responsive as expected. Kindly look into the https://github.com/media query part of your CSS file and fix the issues.
[your-mobile-responsive.webm](https://user-images.githubusercontent.com/26597432/212852973-4f34de9c-3f6b-4250-a86c-91860c75c8ca.webm)

Reply:
As suggested by the previous reviewers, your project is still not responsive. kindly follow the suggestions given by the previous reviewers and make the necessary changes🤗